draft-duke-quic-protected-initial.md

```diff
@@ -416,7 +413,18 @@ as well.
 
 Section 7.8 of {{VERSION-ALIASING}} is also applicable.
 
-## Key Loss and Version Downgrade {#downgrade}
+## Version Downgrade Attack
+
+An attacker might inject Version Negotiation to force the connection to migrate
```

Comment:
This section can be removed: this document has no dependency on VN

Reply:
I disagree. This is uniquely vulnerable to version downgrade, because the entire added value occurs in the Initial Packets, which happen before the endpoints can detect a downgrade. At the very least, I think we need the text about waiting around when there's a VN packet before sending a new Initial.

Reply:
Opened #30